.golangci.example.yml

```diff
@@ -382,3 +382,27 @@ issues:
 
   # Show only new issues created in git patch with set file path.
   new-from-patch: path/to/patch/file
+
+  # Default value is empty string.
+  # Set the default severity for issues. If severity rules are defined and the issues 
+  # do not match or no severity is provided to the rule this will be the default 
+  # severity applied. Severities should match the supported severity names of the 
+  # selected out format.
+  # - Code climate: https://docs.codeclimate.com/docs/issues#issue-severity
+  # -   Checkstyle: https://checkstyle.sourceforge.io/property_types.html#severity
+  # -       Github: https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message
+  severity-default: error
```

Comment:
I think it should be in the nested object similar to:

```yaml
severity:
  default-severity: error
  case-sensitive: false
  rules:
    - ...
```

Reply:
Severity was moved to it's own parent key